tests/test_wave_01.py

```diff
@@ -119,12 +119,13 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
     
-    raise Exception("Test needs to be completed.")
+    # raise Exception("Test needs to be completed.")
+    assert  MOVIE_TITLE_1 == updated_data["watched"][0]["title"]
```

Comment:
You could also assert against the genre and rating key/values in the same way, for more thoroughness!